src/js/shared/application-form.js

```diff
@@ -133,46 +136,52 @@ exports.createApplicationForm = function createApplicationForm(validateFile = tr
 
           callback();
         }
-      ]
-    }),
-    team_apply: fields.boolean({
-      label: 'Are you applying as part of a team?',
-      note: 'We will not process your application until you have been entered into a team via the application form (after submitting this form).',
-      widget: checkboxWidget('Yes, I am applying as part of a team. One of our members will fill out the team application form.'),
-      cssClasses,
+      ],
+      row_units: 'two half',
     }),
-    team_placement: fields.boolean({
-      label: 'If not, would you like us to place you in a team?',
-      note: 'We can suggest a team for you before the event. You can always change this by contacting us.',
-      widget: checkboxWidget('Yes, please place me in a team!'),
+    team: fields.array({
+      label: 'Teams',
+      note: 'If you’re applying as part of a team, we won’t process your application until you’ve been entered into a team using the application form. The team application form can be submitted by any member of the team after every team member has submitted this form.<br>Otherwise we can suggest a team for you before the event. You can always change team by contacting us.',
+      widget: multiCheckboxWidget(),
+      choices: {
+        team_apply: 'I’m applying as part of a team. One team member will fill out the team application form.',
+        team_placement: 'I’m not applying as part of a team. Put me in a team!',
+      },
       validators: [
         (form, field, callback) => {
-          if ((field.data) && (form.fields.team_apply.data)) {
-            callback('We can\'t place you in a team if you are already applying as part of a team!');
+          if ((field.data) && (field.data.length > 1)) {
+            callback('You can’t have an answer and not be sure!');
```

Comment:
This callback message doesn't seem to make sense as there's no notion of being unsure

Reply:
Yup overzealous copy and pasting